src/auth/auth.ts

```diff
@@ -375,21 +375,17 @@ const isNowSignedIn = (oldState: AuthState, state: AuthState) => {
   return oldState.signInStatus !== 'authenticated' && state.signInStatus === 'authenticated';
 };
 
-export const isUserLoaded = (state: AuthState) => {
-  return state.signInStatus === 'userLoaded';
-};
-
-export const hasAcceptedTermsOfService = (state: AuthState): boolean => {
-  return state.termsOfService.permitsSystemUsage ?? false;
+export const isUserInitialized = (state: AuthState) => {
```

Comment:
Super nit: nice to have explicit type declarations for function interfaces.

```suggestion
export const isUserInitialized = (state: AuthState): boolean => {
```